pkg/model/encryption_policy.go

```diff
@@ -0,0 +1,40 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Encryption modes
+const (
+	EncryptNone   = "NONE"
+	EncryptAES128 = "AES128"
+	EncryptAES256 = "AES256"
+)
+
+// EncryptionPolicy contains backup encryption information.
+// @Description EncryptionPolicy contains backup encryption information.
+type EncryptionPolicy struct {
+	// The encryption mode to be used (NONE, AES128, AES256)
+	Mode string `yaml:"mode,omitempty" json:"mode,omitempty" default:"NONE"`
```

Comment:
```suggestion
	Mode string `yaml:"mode,omitempty" json:"mode,omitempty" default:"NONE" enums:"NONE,AES128,AES256"`
```

That would add enums to openapi (and java client)